src/app/services/serviceRoles.js

```diff
@@ -0,0 +1,11 @@
+const { extractPageParam, extractPageSizeParam } = require('../utils');
+const { listServiceRoles } = require('../../infrastructure/applications');
+
+const listServiceRoles = async (req, res) => {
+
+  const serviceRoles = await listServiceRoles(req.params.cid, req.correlationId);
+  return res.send({
```

Comment:
is logging required for cid or correlationId please?

Reply:
I believe so. Logging is absent across much of the codebase. I have added logging now.